addons/mail/static/src/models/chat_window.js

```diff
@@ -566,6 +576,10 @@ registerModel({
             default: false,
             compute: '_computeHasCallButtons',
         }),
+        hasCloseAsBackButton: attr({
+            compute: '_computeHasCloseAsBackButton',
+            default: false,
```

Comment:
there should probably not be a default here then

Reply:
The presence of the feature is handled by boolean value, therefore it's `true` or `false`.
So there must be a default, otherwise `undefined` becomes a value and it's poorly treated as it's falsy.

For me `default` has an important meaning that is clearly not the same as explicitly choosing `true` or `false`.
But this doesn't mean that the `default` should have a different value than `true` or `false`. In fact, this would be a wrong design if we had a different resulting value, because this would mean that the presence of feature is "yes", "no", and "I don't know".

It's not reasonable to not conclude with activation or deactivation of feature in the end, so the resulting value **_must_** be `true` or `false`.

So as a reply to this comment, there should definitely be a default here, and it must be "yes" or "no".

Reply:
But this is a computed field, not a field on which the user should write a value.

Having a default is really just making the compute more complex by introducing clear and having to look into 2 different places to understand what is going to happen.

We can code the compute to always determine the presence of the feature.

Reply:
Every field will become computed fields eventually, so that all of them are as declarative as you can get.
The content of computation matches specs, whether by core design or user customisation.
In both cases, every clear decisions should give proper result, so `true` or `false`. And every cases that we don't really care should show it's a "don't really care", even when the result is `true` or `false`.

IMO, it's very important to see this "we don't care" part, and that's the purpose of `default` for boolean fields.